docs/input.rst

```diff
@@ -141,7 +141,27 @@ still provide the following information:
   ``'NOT_ALIVE_DISPOSED'``), the sample data contains the value of the key that
```

Comment:
Here is the output associated with this PR: https://jenkins-community.rti.com/job/ci/job/Connector-PY/job/rticonnextdds-connector-py-doc/view/change-requests/job/PR-179/Connector_20Documentation/index.html#.